test/javascript/public/test/validateElement.js

```diff
@@ -152,6 +221,58 @@ QUnit.test('Validate when focusouting on customized_field', function (assert) {
   assert.ok(label.parent().hasClass('field_with_errors'))
 })
 
+QUnit.test('Validate when focusouting on date_field', function (assert) {
+  var form = $('form#new_user')
+  var input = form.find('input#user_date_of_sign_up')
+  var label = $('label[for="user_date_of_sign_up"]')
+
+  input.trigger('focusout')
+  assert.ok(input.parent().hasClass('field_with_errors'))
+  assert.ok(label.parent().hasClass('field_with_errors'))
+})
+
+QUnit.test('Validate validations of date_select', function (assert) {
+  var form = $('form#new_user')
+
+  //var label = $('label[for="user_date_of_birth_1i"]')
+  var input_year = form.find('select#user_date_of_birth_1i')
+  var input_month = form.find('select#user_date_of_birth_2i')
+  var input_day = form.find('select#user_date_of_birth_3i')
+
+  input_year.trigger('focusout')
+  assert.ok(input_year.parent().hasClass('field_with_errors'))
+
+  input_month.trigger('focusout')
+  assert.ok(input_month.parent().hasClass('field_with_errors'))
+
+  input_day.trigger('focusout')
+  assert.ok(input_day.parent().hasClass('field_with_errors'))
+
+  // showing validation messages doesnt work well with this.
+  // JS Formbuilder must be customized for these types of fields
+  // to share error message and hide error only when all 3 selects are valid
```

Comment:
This will require attention

Reply:
I know. But I am focused on getting CSV-simple_form work. Not sure if I will have time for this here. I just put it into main, because it was missing here and believed it won't break anything (because time/date_selects wasn't working before anyway)